static/styles/night_mode.scss

```diff
@@ -50,10 +50,13 @@ body.night-mode {
         color: inherit;
     }
 
+    .message_reaction_count {
+        color: hsl(230, 70%, 69%);
+    }
```

Comment:
Can you explain why we need the change to the count color, in addition to the background color?

Reply:
@timabbott  I think I must have changed it earlier while making changes in the colors, and forgot to change it back. I will change it back to as it was, also someone suggested some other changes on the zulipchat channel, so I have tried that as well and posted a screenshot there, so please review that and let me know if that looks better.